src/EA.Weee.Web/Areas/Admin/Controllers/ManageEvidenceNotesController.cs

```diff
@@ -98,10 +104,94 @@ public async Task<ActionResult> ViewEvidenceNoteTransfer(Guid evidenceNoteId)
+
+                return View(model);
+            }
+        }
+
+        [HttpGet]
+        [AuthorizeInternalClaims(Claims.InternalAdmin)]
+        public async Task<ActionResult> VoidTransferNote(Guid transferEvidenceNoteId)
+        {
+            SetBreadcrumb(BreadCrumbConstant.ManageEvidenceNotesAdmin);
+
+            using (var client = apiClient())
+            {
+                var request = new GetEvidenceNoteTransfersForInternalUserRequest(transferEvidenceNoteId);
+
+                var transferNoteData = await client.SendAsync(User.GetAccessToken(), request);
+
+                if (transferNoteData.Type == NoteType.Transfer && transferNoteData.Status == NoteStatus.Approved)
+                {
+                    var model = mapper.Map<ViewTransferNoteViewModel>(new ViewTransferNoteViewModelMapTransfer(transferNoteData.TransferredOrganisationData.Id,
+                       transferNoteData, TempData[ViewDataConstant.TransferEvidenceNoteDisplayNotification]));
+
+                    model.CanVoid = IsInternalAdminUser();
+
+                    return View("VoidTransferNote", model);
+                }
+
+                return new HttpStatusCodeResult(HttpStatusCode.Forbidden, "This note is not a transfer note or has not been approved.");
+            }
+        }
+
+        [HttpPost]
+        [AuthorizeInternalClaims(Claims.InternalAdmin)]
+        [ValidateAntiForgeryToken]
+        public async Task<ActionResult> VoidTransferNote(ViewTransferNoteViewModel model)
+        {
+            SetBreadcrumb(BreadCrumbConstant.ManageEvidenceNotesAdmin);
+
+            using (var client = apiClient())
+            {
+                var request = new GetEvidenceNoteTransfersForInternalUserRequest(model.EvidenceNoteId);
+
+                var result = await client.SendAsync(User.GetAccessToken(), new VoidTransferNoteRequest(model.EvidenceNoteId, model.VoidedReason));
+
+                var transferNoteData = await client.SendAsync(User.GetAccessToken(), request);
+
+                if (transferNoteData.Type == NoteType.Transfer && transferNoteData.Status == NoteStatus.Void)
+                {
+                    model.CanVoid = IsInternalAdminUser();
+
+                    TempData[ViewDataConstant.TransferEvidenceNoteDisplayNotification] = true;
+
+                    model = mapper.Map<ViewTransferNoteViewModel>(new ViewTransferNoteViewModelMapTransfer(transferNoteData.TransferredOrganisationData.Id,
+                       transferNoteData, TempData[ViewDataConstant.TransferEvidenceNoteDisplayNotification]));
+
+                    return RedirectToAction("ViewTransferNote", "ManageEvidenceNotes", new { transferNoteId = model.EvidenceNoteId });
+                }
+            }
+
+            return View("VoidTransferNote", model);
+        }
+
+        [HttpGet]
+        [AuthorizeInternalClaims(Claims.InternalAdmin)]
+        public async Task<ActionResult> ViewTransferNote(Guid transferNoteId)
+        {
```

Comment:
As commented above, we shouldnt need this action should be able reuse ViewEvidenceNoteTransfer

Reply:
@GuidoWorkDev This isnt actioned...